core/domain/user_services.py

```diff
@@ -110,6 +110,9 @@ def __init__(
                 user last edited an exploration.
             profile_picture_data_url: str or None. User uploaded profile
                 picture as a dataURI string.
+            default_dashboard: str|None. The dashboard the user wants.
```

Comment:
Rephrase to: `The default dashboard of the user.`

Reply:
Done.